aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,366 @@
+#  Copyright 2023 Adobe. All rights reserved.
+#  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License. You may obtain a copy
+#  of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under
+#  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+#  OF ANY KIND, either express or implied. See the License for the specific language
+#  governing permissions and limitations under the License.
+
+# Internal Library
+
+import aepp
+from aepp import flowservice
+from aepp import destinationinstanceservice
+import time
+import logging
+from typing import Union
+from aepp.utils import Utils
+from .configs import ConnectObject
+from tenacity import Retrying
+from tenacity import retry_if_result
+from tenacity import wait_fixed
+from tenacity import stop_after_attempt
+from dataclasses import dataclass,asdict,field
+
+@dataclass
+class Flow:
+    name: str = None
+    flowSpec: dict = field(default_factory=dict)
+    sourceConnectionIds: list = field(default_factory=list)
+    targetConnectionIds: list= field(default_factory=list)
+    scheduleParams: dict = field(default_factory=dict)
+    transformations: list = field(default_factory=list)
+
+    def __init__(self,
+                 name: str,
+                 flow_spec: dict,
+                 source_connection_id: str,
+                 target_connection_id: str,
+                 schedule_params: dict,
+                 transformations: dict
+                 ):
+        self.name = name
+        self.sourceConnectionIds = [source_connection_id]
+        self.targetConnectionIds = [target_connection_id]
+        self.flowSpec = flow_spec
+        self.scheduleParams = schedule_params
+        self.transformations = transformations
+
+class ExportDatasetToCloudStorage:
+    """
+    A class for exporting dataset to cloud storage functionality
+    Attributes
+    ----------
+    flow_conn : module
+        flow service connection
+    dis_conn : module
+        destination instance service connection
+    username : str
+        the log in for the os
+    logger: module
+        class logger
+    """
+
+    flow_conn = None
+    dis_conn = None
+    username = None
+    logger = None
+    DLZ_CONNECTION_SPEC_ID = "10440537-2a7b-4583-ac39-ed38d4b848e8"
```

Comment:
SInce these are DLZ related, does that mean this class can only export to DLZ? I think it should support any type of destination, otherwise you should probably rename the class `ExportDatasetToDLZ` or something like that.

Reply:
@cmenguy, each destination will need customized work around its payload.  will rename to ExportDatasetToDLZ.